justfile

```diff
@@ -0,0 +1,7 @@
+repository := "mesosphere"
+include_file := justfile_directory() / ".include-airgapped"
+exclude_file := justfile_directory() / ".exclude-airgapped"
+
+release-oci tag tmp_dir=`mktemp --directory`:
+    rsync --info=name --archive --recursive --files-from={{ include_file }} --exclude-from={{ exclude_file }} {{ justfile_directory() }} {{ tmp_dir }}
+    echo "${DOCKER_PASSWORD}" | oras push --password-stdin --user "${DOCKER_USERNAME}" --verbose {{ repository }}/kommander-applications:{{ tag }} {{ tmp_dir }}
```

Comment:
`mktemp` returns an absolute path to the directory, and `oras push` will fail unless the `--disable-path-validation` is provided. If the flag is set, `oras push` will not complain but it will preserve the whole directory structure, starting from `/`, replicating what the `tmp_dir` is pointing to, so we might want to call `oras push` from the `tmp_dir` directly, e.g. `oras push <args> .`